src/timeline/js/controllers.js

```diff
@@ -1092,7 +1092,7 @@ App.controller("TimelineCtrl", function ($scope) {
   };
 
   // Search through clips and transitions to find the closest element within a given threshold
-  $scope.getNearbyPosition = function (pixel_positions, threshold, ignore_ids) {
+  $scope.getNearbyPosition = function (pixel_positions, threshold, ignore_ids=[]) {
```

Comment:
```suggestion
  $scope.getNearbyPosition = function (pixel_positions, threshold, ignore_ids={}) {
```
Looking at the only call to this function in the codebase (`functions.js` line ~313, IIRC?), when it IS passed in `$ignore_ids` will be an object, not a list. (And on line 1129 here, `.hasOwnProperty()` is used on it, reinforcing that.)

An empty list might work as well (lists are objects, right?) but might as well keep consistent.

Reply:
+1 on this suggestion